Defined process for handling misscrambles (#1266)
thewca/wca-regulations: wca-guidelines.md

- 4d++) [ADDITION] It is permitted for the puzzle to change its orientation when it is moved from the scrambler to the solving station, as long as no one is attempting to influence the randomness of the orientation (see [Regulation A2e1](regulations:regulation:A2e1)).
- 4f+) [RECOMMENDATION] The WCA Delegate should generate sufficient scramble sequences for the entire competition ahead of time, including spare scramble sequences for extra attempts.
- 4f++) [REMINDER] If the WCA Delegate generates any additional scramble sequences during the competition, the scramble sequences must be saved.

> - 4g+) [RECOMMENDATION] After verifying the puzzle is scrambled correctly, the scrambler may pass the puzzle to another scrambler to verify. In this case, only one of the scramblers is required to sign the scorecard.

Reviewer: ```suggestion
- 4g+) [RECOMMENDATION] After verifying the puzzle is scrambled correctly, the scrambler may pass the puzzle to another scrambler to verify. In this case, both scramblers should sign the scorecard.
```
Since this provides better accountability for both scramblers and clearer incident tracking.

> - 4g+) [RECOMMENTATION] After verifying the puzzle is scrambled correctly, the scrambler may pass the puzzle to another scrambler to verify as well. In this case, only one of the scramblers is required to sign the scorecard.

Author: ```suggestion
- 4g+) [RECOMMENDATION] After verifying the puzzle is scrambled correctly, the scrambler may pass the puzzle to another scrambler to verify as well. In this case, only one of the scramblers is required to sign the scorecard.
```

> - 4g+) [RECOMMENDATION] After verifying the puzzle is scrambled correctly, the scrambler may pass the puzzle to another scrambler to verify as well. In this case, only one of the scramblers is required to sign the scorecard.

Reviewer: ```suggestion
- 4g+) [RECOMMENDATION] After verifying the puzzle is scrambled correctly, the scrambler may pass the puzzle to another scrambler to verify. In this case, only one of the scramblers is required to sign the scorecard.
```
To align with @Rouxles feedback on the alternate proposal [here](https://github.com/thewca/wca-regulations/pull/1267/files).

- 11e1+) [REMINDER] The extra attempt must be scrambled using an unmodified scramble sequence generated by an official scramble program (see [Regulation 4f](regulations:regulation:4f), [Guideline 4f+](guidelines:guideline:4f+)).
- 11e2+) [ADDITION] If an extra attempt itself leads to an additional extra attempt, the competitor should keep doing extra attempts until there is a valid extra attempt to replace the original regularly numbered attempt.
- 11e2++) [EXAMPLE] Suppose a competitor has 5 regularly numbered attempts, and #2 leads to extra attempts. The competitor should do extra attempts until there is a valid extra attempt to replace #2. Suppose later another attempt leads to extra attempts, the competitor should proceed in a similar way as for #2.
- 11i1+) [RECOMMENDATION] Competitors expected to meet these criteria should have their scrambles checked by a second scrambler before being sent out (see [Guideline 4g+](guidelines:guideline:4g+)).

> - 11i2+) [RECOMMENDATION] If the WCA Delegate is unsure whether or not the incorrect scramble is significantly easier than the original scramble, a provisional extra attempt should be granted.

Author: ```suggestion
- 11i2+) [RECOMMENDATION] If the WCA Delegate is unsure whether or not the incorrect scramble provided a significantly less fair outcome than the original scramble, a provisional extra attempt should be granted.
```
